.travis.yml

```diff
@@ -2,8 +2,8 @@ sudo: false
 language: go
 
 go:
- - 1.5.4
- - 1.6.2
+ - 1.6
```

Comment:
The former is what this scheme of providing a version generally implies.
Couldn't find anything explicit in the Travis docs via googling though.